src/lsp/cobol_common/visitor.ml

```diff
@@ -103,7 +111,13 @@ module Fold = struct
 
   (** [handle fold continue node acc] first calls [fold node acc], and then
       behaves according to the action returned. *)
-  let handle (fold: 'x -> 'a -> 'a action) ~(continue: 'x -> 'a -> 'a) n x =
+  let handle :
```

Comment:
I'd say the single-line above was much more readable (maybe just missing `(n: 'x) (x: 'a)` if you really want those).